ts/Core/Axis/Axis.ts

```diff
@@ -989,42 +989,69 @@ class Axis {
             return x;
         }
 
-        const evt: Event = {
+        const evt: Partial<Event&Axis.PlotLinePathOptions> = {
```

Comment:
When I think about it, I think it should be just `Partial<Axis.PlotLinePathOptions>`Since there are no `Event`'s attributes, but your call.